privacyidea/lib/config.py

```diff
@@ -451,6 +452,9 @@ def get_token_class(tokentype):
             tokenclass = tclass
             break
 
+    if tokenclass is None:
+        raise ParameterError(u"Invalid tokentype provided. ttype: {}".format(tokentype.lower()))
```

Comment:
@Happy2C0de  It looks as if your change is too low level here, that causes several tests to fail!

Reply:
@cornelinux 
I moved the error handling to `ttype.py`.
LGTM